src/SymfonyCache/Tag/ManagerInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace FOS\HttpCache\SymfonyCache\Tag;
+
+interface ManagerInterface
+{
+    /**
+     * Return the concrete cache paths for the given tag.
+     *
+     * @param string $tag
+     * @return string[]
+     */
+    public function getPathsForTag($tag);
```

Comment:
This is a utility method really for testing. Could be removed from the interface.